zone/guild.cpp

```diff
@@ -317,13 +362,14 @@ void EntityList::SendGuildMembers(uint32 guild_id) {
 	while (it != client_list.end()) {
 		Client *client = it->second;
 		if (client->GuildID() == guild_id) {
-			client->SendGuildMembers();
+			//client->SendGuildMembers();
```

Comment:
Commented? Is there supposed to be a TODO here ?

Reply:
This was part of the work to not have to send an entire guild list for every action (member level update, public note, alt, banker, etc.)  I removed this.  I have to check this more closely as I am unsure if I re-added all the TI/UF versions.  This should stay open while I check.

Reply:
Sounds good